hbase-rest/src/main/java/org/apache/hadoop/hbase/rest/model/ScannerModel.java

```diff
@@ -18,22 +17,19 @@
  */
 package org.apache.hadoop.hbase.rest.model;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
```

Comment:
Should be org.apache.hbase.thirdparty.com.fasterxml.jackson...

Reply:
Nope, because I don't relocate Jackson, just the `jackson-jaxrs-json-provider` and things related specifically to jaxrs. The other PR uses an `includes` list, not excludes, and only relocates the package prefix `com.fasterxml.jackson.jaxrs`. Transitive dependencies of the two artifacts it relocates are preserved.